dll/cpl/appwiz/lang/zh-TW.rc

```diff
@@ -47,15 +48,15 @@ FONT 9, "新細明體", 0, 0, 0x0
 BEGIN
     LTEXT "輸入捷徑名稱(&T)：", -1, 120, 15, 150, 10
     EDITTEXT IDC_SHORTCUT_NAME, 120, 35, 150, 15, ES_LEFT
-    LTEXT "點選 [完成] 創建捷徑", -1, 118, 97, 178, 17
+    LTEXT "按 [完成] 建立捷徑", -1, 118, 97, 178, 17
 END
 
 IDD_CONFIG_STARTMENU DIALOGEX 0, 0, 212, 200
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "刪除捷徑/資料夾"
-FONT 8, "新細明體"
+CAPTION "刪除捷徑／資料夾"
+FONT 9, "新細明體"
```

Comment:
How does this looks like after the change?
Does this looks better? Or any reason why using ```／``` instead of ```/``` 😅

Reply:
well the difference between `／` and `/` is just 全形 and 半形.
![全形 and 半形](https://user-images.githubusercontent.com/48596398/90719026-e4121580-e2e5-11ea-8ecc-f5eb2d2c2339.png)

BTW I am going to revert the change because of the consistency, since  `/` is more often to be used in other files. 
Also, the use of `／` is just my personal habit. 🙈
PS: I should test the change before submitting pull request 😅

Reply:
I prefer add two spaces around ```/``` like this:
```刪除捷徑 / 資料夾```
it's up to you, you can decide it 🤣